### ISSUE TITLE
[ Question ] If a module update, will it reset all the css and html files inside them?

### ISSUE THREAD

**Issue by FireFoXz09** [RCVolus/league-prod-toolkit#155]
If you update modules, will it reset the html or css inside? For example, I modify the layout or emplacement of the plating overlay (to appear on the bottom of the screen) or the inhibitor overlay to be elsewhere on my screen. Will this data be reset? The wiki on customisation is confusing and don't seem to be up to date or to clarify this part...

**Comment by sdomi4** [comment]
Yes, a module update overwrites the entire module. If you modified any of the files directly, these changes will be lost.
Customisation is intended through themes, which will not be affected by any updates.

The wiki isn't very detailed on customisation at the moment, I will leave this issue open so I remember to expand it :)
